### ISSUE TITLE
Generic.WhiteSpace.ScopeIndent - indent follow up lines of multiline statement

### ISSUE THREAD

**Issue by MartinMystikJonas** [squizlabs/PHP_CodeSniffer#3244]
**Describe the bug**

**Code sample**
```php
        $statusString =
            "Lorem ipsum dolor sit amet." .
            "Lorem ipsum dolor sit amet..";
```

**Custom ruleset**
```xml
<?xml version="1.0"?>
<ruleset name="My Custom Standard">
  <rule ref="Generic.WhiteSpace.ScopeIndent">
    <properties>
      <property name="exact" value="true"/>
    </properties>
  </rule>
</ruleset>
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php ...`
3. See error message displayed
```
PHPCS output here
```

**Expected behavior**
I would like to be able to instruct this rule to indent following lines of multiline statements to have +1 indent not same indent as first line. Or at least allow not exact matching on following lines.

**Versions (please complete the following information):**
 - OS: Linux
 - PHP: 7.4
 - PHPCS: 3.5.8

**Comment by gsherwood** [comment]
This isn't a bug - exact matching means it has to be exact. I never recommend people use this setting across their entire code base as it's almost never what you want.

**Comment by rodrigoprimo** [comment]
@jrfnl, based on the original maintainer's feedback, I believe this issue can be closed.
